environments/servlet/tests/jetty/pom.xml

```diff
@@ -314,9 +314,9 @@
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-surefire-plugin</artifactId>
                         <configuration>
-                            <excludes>
-                                <exclude>%regex[.*servlet/test/(?!se).*]</exclude>
-                            </excludes>
+                            <includes>
```

Comment:
I'm not going to include `WeldSeBuilderTest` quite yet - it's failing locally for other reasons which probably warrants its own PR.